src/os/bluestore/NVMEDevice.cc

```diff
@@ -1090,11 +1095,10 @@ int NVMEDevice::aio_read(
   Task *t = new Task(this, IOCommand::READ_COMMAND, off, len);
 
   bufferptr p = buffer::create_page_aligned(len);
-  pbl->append(p);
+  pbl->append(t->bl);
```

Comment:
Could you explain why use std:move() sice at this time, there is no data copy yet. Also in kernelDevice.cc, it also directly use append.